src/Jackalope/Version/VersionHandler.php

```diff
@@ -103,14 +107,20 @@ public function checkinItem($path)
         }
 
         if ($node->isModified()) {
-            throw new InvalidItemStateException(sprintf(
-                'Node "%s" contains unsaved changes',
-                $path
-            ));
+            throw new InvalidItemStateException(
+                sprintf(
+                    'Node "%s" contains unsaved changes',
+                    $path
+                )
+            );
         }
 
+        /** @var NodeInterface $baseVersionNode */
```

Comment:
afaik this is already known from the return type annotation on ObjectManager::getNodeByIdentifier.